python/lsst/ts/xml/data/sal_interfaces/ATWhiteLight/ATWhiteLight_Events.xml

```diff
@@ -119,8 +119,8 @@
   </Enumeration>
   <Enumeration>
     LampControllerState_Unknown=0,
-    LampControllerState_Off=1,
-    LampControllerState_StandbyOrOn=2,
+    LampControllerState_Standby=1,
```

Comment:
Not related to this PR but in `LampBasicState` the value 3 is skipped. See lines 99 to 106 of this file.

Reply:
Good catch! Fixed